Xamarin.Forms.Core/Touch.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+
+namespace Xamarin.Forms
+{
+	public class Touch
```

Comment:
public class TouchGestureEventArgs  : EventArgs

Reply:
This is gesture information, this is not an event.